### PR TITLE
Add fixes to script of PR

### PR DESCRIPTION
**Pull request by hadrich-hatem**
| Questions     | Answers
| ------------- | -------------------------------------------------------
| Description?  | This PR allows to add some fixes for all scenarios of PRs. 
| Fixed PR? | /
| How to test?  | Run the script: TEST_PATH=npm run PR -- --URL='http://FrontOfficeURL'

### CODE REVIEW

File: test/mocha/campaigns/PR/10169.js [hunk 1]
```diff
@@ -20,6 +22,15 @@ let productData = {
 
 scenario('PR-10169: Check that the images is well displayed in the edit combination', () => {
   authentication.signInBO('10169');
+  scenario('Close the onboarding modal if exist ', client => {
```
Comment (fatmaBouchekoua) [inline test/mocha/campaigns/PR/10169.js]: You can add a function to close the onboarding modal, and you can use it for all files

File: test/mocha/campaigns/PR/9206.js [hunk 2]
```diff
@@ -1,35 +1,46 @@
...
-  scenario('Search then check that the SQL query was well created', client => {
-    test('should search for the created "SQL query"', () => client.searchByValue(SqlManagerPage.filter_sql_query_name_input, SqlManagerPage.filter_search_button, 'ps_shop' + dateTime));
-    test('should check the appearance of created "SQL query" in the table', () => client.checkTextValue(SqlManagerPage.sql_query_name.replace('%S', 1), 'ps_shop' + dateTime, 'equal', 2000));
-  }, 'common_client');
-  scenario('Delete the created SQL query', client => {
-    test('should click on "Dropdown" button then click on "Delete" action', async () => {
-      await client.waitForAndClick(SqlManagerPage.sql_query_dropdown_button.replace('%S', 1));
-      await client.confirmationDialog();
-      await client.waitForAndClick(SqlManagerPage.sql_query_dropdown_action.replace('%S', 1).replace('%ACTION', 'delete'), 2000);
+    test('should close the "Onboarding" if exists', async () => {
```
Comment (fatmaBouchekoua) [inline test/mocha/campaigns/PR/9206.js]: you have closed the onboarding modal twice. Is there a reason ?

Comment (hadrich-hatem) [reply test/mocha/campaigns/PR/9206.js]: I have added a test to close the onBoarding modal and the other one to stop the onBoarding.

**Comment by hadrich-hatem** [comment]
@mbadrani this PR is ready to be merged.

**Pull request merged**